docs/development/running-the-dev-environment.md

```diff
@@ -1,154 +1,51 @@
-# Running the dev environment
+# Running the Dev Environment
```

Comment:
Would it make sense to combine this document with `development-environment.md`? I don't see much benefit in these two separate docs.